internal/backends/winit/lib.rs

```diff
@@ -224,6 +229,17 @@ impl BackendBuilder {
         self
     }
 
+    /// Configures this builder to enable or disable the default menu bar.
+    /// Set this to false if you're providing your own menu bar.
+    /// Note that if the Slint code declares a `MenuBar`, then that will
+    /// override an application provided menu.
+    #[must_use]
+    #[cfg(all(muda, target_os = "macos"))]
+    pub fn with_default_menu_bar(mut self, enable: bool) -> Self {
```

Comment:
The docs should probably should state that this defaults to `true`

Maybe it should be named `with_no_default_menu_bar(self)` but i understand it's better to keep it consistant with winit, so it's probably fine the way it is.

Reply:
No, I think you have a good point. We should pick names that we're comfortable with, and your suggestion is better.

Reply:
Agreed. I've changed the docs.